ms/MSSel/MSArrayParse.h

```diff
@@ -99,7 +99,7 @@ class MSArrayParse : public MSParse
   const TableExprNode *selectRangeGTAndLT(const Int& n0, const Int& n1);
   const TableExprNode *selectRangeGEAndLE(const Int& n0, const Int& n1);
   const TableExprNode *selectArrayIds(const Vector<Int>& arrayids);
-  inline const TableExprNode *selectArrayIds() {return selectArrayIds(parsedIDList_p);}
+  inline const TableExprNode *selectArrayIds() {return selectArrayIds(Vector<int>(parsedIDList_p));}
```

Comment:
Should be Vector<Int> (with capital I)
Same in next 2 files.

Reply:
Thanks, corrected.